app/src/main/java/com/example/mvvmcodebase/data/model/Cocktail.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.mvvmcodebase.data.model
+
+data class Cocktail(val strDrink: String, val strCategory: String, val strDrinkThumb: String)
```

Comment:
Để việc đặt tên key của các trường trong response trả về không bị phụ thuộc với class định nghĩa dưới app, sử dụng serializedname 
```suggestion
data class Cocktail(
    @SerializedName("strDrink")
    val drink: String, 
    val strCategory: String, 
    val strDrinkThumb: String
)
```